### ISSUE TITLE
Change url from http://opensource.org/ to https://spdx.org/licenses/

### ISSUE THREAD

**Issue by SimenB** [npm/normalize-license-data#1]
Seeing as npm merged https://github.com/npm/npm/pull/8179, maybe link to SPDX's licenses, as all of them are "accepted", while http://opensource.org/ has a quite limited selection?

**Comment by SimenB** [comment]
@zeke @kemitchell @othiym23  Multi-license is broken for the website. See for example this: https://www.npmjs.com/package/JSONStream. Clicking the license 404s.

Not sure if a switch to e.g. spdx.org should be done, or update https://github.com/npm/oss-license-name-to-url to understand multiple licenses?

Something a bit more human readable than `(MIT OR Apache-2.0) license` as well, perhaps?

**Comment by kemitchell** [comment]
@simenb, I just helped @dominictarr add that license expression to JSONStream. Good catch!

This is almost certainly a www issue. Might you open an issue there and ping me? I will try to get a PR out this weekend.

For the record, yes, I think we should link to licenses on SPDX.org.

**Comment by kemitchell** [comment]
@SimenB, unless you object, I think this issue can close. The action's at npm/newww#992.

**Comment by SimenB** [comment]
It's this module (I think? :laughing:) providing the URL, but I'm sure it's fine tracking it over there.

**Comment by kemitchell** [comment]
No, I think you're right. Would you mind opening again? I may need to PR both.
